src/tools/base/BaseBrushTool.js

```diff
@@ -50,7 +50,9 @@ class BaseBrushTool extends BaseTool {
    *
    * @abstract
    * @param {Object} evt - The event.
+   * @returns {void}
    */
+  // eslint-disable-next-line no-unused-vars
   renderBrush(evt) {
```

Comment:
Thank you for using `// eslint-disable-next-line no-unused-vars` here.
It's really helpful to know, in these situations, what is passed to the override-able class methods

Reply:
🤔 I wonder if there is an eslint rule that covers, or creates an exception, for cases like this.

Reply:
Might be cleaner to add `// eslint no-unused-vars: 0` before the block of `abstract` methods, then after the block turn the rule back on with `// eslint no-unused-vars: 1`. This will remove the bloat at the top of each abstract method, and also forces us to keep the abstract methods nicely organized together at the top of the class.

Reply:
Maybe it's cleaner to document this with [`@abstract`](http://usejsdoc.org/tags-abstract.html) and an `@example` which lists the arguments which will be passed, rather than having code that never runs.

Reply:
@mikehazell, that sounds like a great solution. Does the linter yell at us for defining parameters not included in the signature? If not, then I like this approach.

Reply:
Yeah if the linter is happy with that I also love that solution.